include/flatbuffers/stl_emulation.h

```diff
@@ -41,15 +41,15 @@
   #include <optional>
 #endif
 
-// The __cpp_lib_span is the predefined feature macro.
-#if defined(FLATBUFFERS_USE_STD_SPAN)
-    #include <span>
-#elif defined(__cpp_lib_span) && defined(__has_include)
-  #if __has_include(<span>)
-    #include <array>
-    #include <span>
-    #define FLATBUFFERS_USE_STD_SPAN
+#ifndef FLATBUFFERS_USE_STD_SPAN
+  #if defined(__cplusplus) && __cplusplus >= 202002L
```

Comment:
Can you leave a doc comment here, perhaps with a link (https://en.cppreference.com/w/cpp/utility/feature_test)

Reply:
Sure. Like this?